deepchem/models/torch_models/grover.py

```diff
@@ -23,14 +23,6 @@ class GroverEmbedding(nn.Module):
         the number of message passing blocks.
     num_head: int
         the number of attention heads.
-    embedding_output_type: str
```

Comment:
To confirm, we are removing embedding_output_type since it is not described in the Grover paper?

Reply:
yes